tests/layer_tests/pytorch_tests/test_log_softmax.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import torch
+import torch.nn.functional as F
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+class aten_log_softmax(torch.nn.Module):
+    def __init__(self, dim, dtype) -> None:
+        super().__init__()
+        self.dim = dim
+        self.dtype = dtype
+
+    def forward(self, input_tensor):
+        return F.log_softmax(input_tensor, dim = self.dim, dtype = self.dtype)
+
+class TestLogSoftmax(PytorchLayerTest):
+    def _prepare_input(self):
+        if self.input_dtype == torch.float:
+            self.input_tensor = np.random.randn(5, 9, 7)
+        else:
+            self.input_tensor = np.random.randint(-100, 100, (5, 9, 7))
+        return (self.input_tensor,)
+
+    @pytest.mark.parametrize(["input_dtype", "convert_dtype"], [
+        # For a tensor with element dtype int, it is necessary to 
```

Comment:
Please take a look on #17418 PR where same issue was fixed on operation level. Basically you need to align input type to fp32

Reply:
Done, although I couldn't add a test for it since pytorch crashes when input dtype == int & convert dtype == None

Reply:
Maybe I misunderstood your comment then. What if input is int, and output type is int, does it work? If pytorch doesn't do automatic int to float conversion then this change to align types is not really needed

Reply:
Convert_dtype happens before log_softmax, it's not the output type - in your example, pytorch raises `RuntimeError: "log_softmax_lastdim_kernel_impl" not implemented for 'Int'`

Reply:
Please remove align dtype if pytorch itself do not support such case. We may return it in future if pytorch will be supporting this, but now it is just an extra step that does nothing